libs/platform/experience/src/services/layout/plugins/properties/bleed/index.ts

```diff
@@ -1,2 +1 @@
 export * from './bleed-layout.model';
```

Comment:
```suggestion
/** 
 * Do NOT export any plugin files, as we'd break lazy loading otherwise. 
 * All plugin artefacts are publicly available in a static entry point, to allow for customisations.
 */
export * from './bleed-layout.model';
```

Or something like that. Can we do that in all those plugins, or if you think we can do it in a central place it might also work nice.

Reply:
added it here
`layout.static.ts`

Reply:
What about
- moving the note to `static.ts`
- introduce `static.ts` barrel files that we use to export includes to above layers. This will also make it more apparent on what we're doing